packages/http-client-csharp/package.json

```diff
@@ -23,8 +23,8 @@
   },
```

Comment:
can't comment above but I think you will need to change the main and exports entries